src/parser/jobs/whm/Assize.js

```diff
@@ -0,0 +1,82 @@
+import React, {Fragment} from 'react'
+
+import {ActionLink} from 'components/ui/DbLink'
+import ACTIONS from 'data/ACTIONS'
+import STATUSES from 'data/STATUSES'
+import Module from 'parser/core/Module'
+import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {Suggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+
+
+const WASTED_USES_MAX_MINOR = 0 //a single lost assize is worth not being hidden as a minor issue
+const WASTED_USES_MAX_MEDIUM = 2
+
+//uses the benison code for now, but should also check healing efficiency
+export default class Assize extends Module {
+	static handle = 'assize'
+	static dependencies = [
+		'checklist',
+		'combatants',
+		'cooldowns',
+		'enemies',
+		'invuln',
+		'suggestions',
+	]
+
+	_lastUse = 0
+	_uses = 0
+	_totalHeld = 0
+
+	constructor(...args){
+		super(...args)
+
+		const _filter = {
+			by: 'player',
+			abilityId: [ACTIONS.ASSIZE.id],
+		}
+		this.addHook('cast', _filter, this._onApplyBenison)
+		this.addHook('complete', this._onComplete)
+	}
+
+	_onApplyBenison(event){
+		this._uses++
+		if(this._lastUse === 0) { this._lastUse = this.parser.fight.start_time }
+
+		const _held = event.timestamp - this._lastUse - (ACTIONS.ASSIZE.cooldown * 1000)
+		if (_held > 0) {
+			this._totalHeld += _held
+		}
+		//update the last use
+		this._lastUse = event.timestamp
+	}
+
+	_onComplete(){
+		if (this._uses === 0){
```

Comment:
I feel that this split between 0 and >0 is a little redundant, but I'm not 100% on that. Sure it can't be combined into one suggestion?